vid/Cargo.toml

```diff
@@ -38,7 +38,7 @@ ark-ed-on-bls12-377 = "0.4.0"
 ark-ed-on-bls12-381 = "0.4.0"
 ark-ed-on-bn254 = "0.4.0"
 criterion = "0.5.1"
-jf-pcs = { git = "https://github.com/EspressoSystems/jellyfish", tag = "0.4.5", features = ["test-srs"] }
+jf-pcs = { path = "../pcs", features = ["test-srs"] }
```

Comment:
Use path b/c some updates are not reflected in the github yet.

Reply:
Do you intend to switch back before merge?

Reply:
i don't think he can. One of the perils of separate versioning is multi-step upgrade here. 
changing back need another PR: a new tag will be added once this PR is merged, then next one would switch back.